.jenkins/pipelines/pr.Jenkinsfile

```diff
@@ -96,6 +98,8 @@ pipeline {
                         returnStdout: true,
                         script: "git log --max-count=1 --pretty=format:'%H'"
                     ).trim()
+                    // Set pull request id for standalone builds
+                    PULL_REQUEST_ID = params.PULL_REQUEST_ID
```

Comment:
why?

Reply:
This is so that PULL_REQUEST_ID can be set via a manual parameter (e.g. manual PR builds), or automatically as part of the inherent properties of Jenkins multibranch pipelines (triggered automatically via GitHub PRs). Then, PULL_REQUEST_ID is passed to the standalone pipelines so the right PR is checked out on those.